src/amo/components/RatingManager/index.js

```diff
@@ -116,9 +130,9 @@ export class RatingManagerBase extends React.Component<InternalProps, State> {
     };
 
     if (userReview) {
-      log.info(`Editing reviewId ${userReview.id}`);
+      log.debug(`Editing reviewId ${userReview.id}`);
```

Comment:
I realized the logging in this component is a bit noisy. We don't really filter out debug/info but the messages are really just for debugging so I switched them in case we want to filter it out later.